1.0/resources/compatibility.md

```diff
@@ -5,26 +5,27 @@ shortname: Resources
 title: Browser Compatibility
 ---
 
-Unlike other frameworks, **{{site.project_title}} is intended to work in the _latest
-version_ of "evergreen" browsers**. That is, Chrome, Firefox, IE, Opera and Safari. 
-This means that your favorite browser might not be supported. 
+> **{{site.project_title}} 1.0 works in all _latest version_ "evergreen" browsers (Chrome, Firefox, IE, Opera, and Safari)**.
 
-The features required by {{site.project_title}} are not supported natively in 
-all of the evergreen browsers, but a set of web components polyfills are available
-that **do** run in the evergreen browsers and provide the support {{site.project_title}}
-requires.
+---
+
+## Browser support
+
+The features used by {{site.project_title}} are not (yet) supported natively in 
+all browsers. For web components support, {{site.project_title}}
+uses the [polyfills](http://webcomponents.org/polyfills/) from [webcomponents.js](http://webcomponents.org). They're lightweight, work well, and provide the feature support {{site.project_title}} requires.
 
-## Native browser support
+With the polyfills, {{site.project_title}} works in these browsers:
 
 <table>
 <thead>
-  <tr><th></th><th>Chrome</th><th>Firefox</th><th>IE&nbsp;10+*</th><th>Safari 8+</th><th>Chrome (Android)</th><th>Safari (iOS&nbsp;8.1)</th></tr>
+  <tr><th></th><th>Chrome</th><th>Firefox</th><th>IE&nbsp;10+/Edge</th><th>Safari 8+</th><th>Chrome (Android)</th><th>Safari (iOS&nbsp;8.1)</th></tr>
 </thead>
 <tr>
   <td class="feature-title"><a href="http://www.html5rocks.com/en/tutorials/webcomponents/template/">Template</a></td>
```

Comment:
Wonder if we even need to list the different features here since it's all the same?

But open to just merging as is.